Code/Engine/Foundation/Types/Implementation/Variant_inl.h

```diff
@@ -20,119 +20,119 @@ EZ_ALWAYS_INLINE ezVariant::ezVariant(ezVariant&& other) noexcept
   MoveFrom(std::move(other));
 }
 
-EZ_ALWAYS_INLINE ezVariant::ezVariant(const bool& value)
```

Comment:
Should we allow `value` as special name similar to `lhs` ?

Reply:
This would be an exception that's probably ok.